protobuf-kafkaconnect-converter/src/test/proto/PrimitiveTypesSyntax2.proto

```diff
@@ -1,3 +1,5 @@
+//syntax = "proto2";
```

Comment:
Why are we commenting this out?

Reply:
I don't remember making this change, reverted